src/teleop/gui/src/components/DriveMoteusStateTable.vue

```diff
@@ -0,0 +1,180 @@
+<template>
```

Comment:
Maybe @jnnanni has different opinion but what do you think about deleting the DriveMoteusStateTable and just keeping the MoteusStateTable? They listen to different topics anyway

Reply:
Forgot to respond to this but basically we wanted the drive one to be formatted differently where it lines up with the drivetrain. So we made it a different component.